README.md

```diff
@@ -1 +1,56 @@
 # Canopy Frontend RFC
+The Canopy Frontend RFC (Request for Comments) is a git repository where the technologies and processes that apply to all
+frontend squads at Canopy are described.
+
+## Purpose
+1. Provide a way to enact change in Canopy's frontend that impacts all projects and squads.
+2. Clarify what things we are aligned on across the entire front-end, so that everything else can be decided by the members of each squad.
+
+## What this isn't
+1. An infringement on the autonomy of squads. The idea is that if we're aligned on the few important things, we can be autonomous on everything else.
+2. Something oppressively prescriptive.
+3. A technical review process where people get feedback on difficult problems they're facing. Although important, the RFC is not how
+   technical reviews happen.
+
+## Signs that this is working
+1. The content is up-to-date and people care about it.
+2. Things are removed when no longer relevant or correct.
+3. We change the process when it's annoying or doesn't work well.
+4. Ideas have a safe place to develop before being criticized.
+
+## Signs that this isn't working
+1. This blocks people from getting stuff done.
+2. Ivory tower gatekeeping.
+3. Changes are discouraged instead of encouraged.
+4. "Well that's just the way it is."
+5. The process is heavy, bureaucratic, or long.
+6. Some *other* process is being used to achieve the same thing.
+
+## What to do if the RFC isn't working
+Propose a change to the process!
+
+## Process for change
+**Anyone** at Canopy can propose a change.
+
+<img src="https://mk0radicalcandov3r1t.kinstacdn.com/wp-content/uploads/2017/02/gsd-wheel.png" width="300" />
+
+### Step 1 - Idea
+During the idea phase, we should encourage instead of critique. A person can use the #frontend-rfc slack channel
+to discuss, clarify, and foster an idea. We should examine with an open mind the pros and cons, options, and implementation.
+
+Another part of the idea phase is asking the question "Should this be autonomous to the squads or enforced consistently in all squads?"
+
+### Step 2 - Proposal
+A proposal is a pull request to this git repository. Here, an idea is debated and discussed by anyone who wants to participate. Reviewers can make comments
```

Comment:
Should there be a naming convention for these? React RFCs have a number prefix

Reply:
Good question -- do you think it would help? The number at the beginning of the proposal would probably make the default sort order be chronological? Which maybe would be helpful?

Reply:
Or do we not care so much about a chronological view/sorting?

Reply:
I may not fully understand, but I always thought that the PR number would be the number.

But thinking about it, I think Gitlab has different numbers for MRs and issues, whereas Github uses the same incremental number for both? 🤷‍♀️

Reply:
Here's a link for context: https://github.com/reactjs/rfcs/tree/master/text. React puts the pull request number into the name of the file itself.

Reply:
Another potential benefit would be in case there are two different ideas for the same topic. Using a unique identifier would avoid duplicate names (not sure how often that would happen...but it might)

Reply:
you've convinced me. Let's try it out. I'll update my code.